daisy_workflows/image_build/debian/build_fai.py

```diff
@@ -38,13 +38,10 @@
 
 def main():
   # Get Parameters.
-  build_date = utils.GetMetadataAttribute(
```

Comment:
build_date is actually used by our workflows as a common field to get the same format for everything.

Reply:
do you use `build_date` in the tarball filename? should I do that?

Reply:
Yeah, we actually use it everywhere- it gets plumbed through from our automation pipeline as well. It can either be a UTC timestamp or a YYYYMMDD timestamp.

Reply:
ok, I'll fix it.